includes/data-stores/class-wc-product-data-store-cpt.php

```diff
@@ -1326,13 +1326,54 @@ public function get_products( $args = array() ) {
 	public function search_products( $term, $type = '', $include_variations = false, $all_statuses = false ) {
 		global $wpdb;
 
-		$like_term     = '%' . $wpdb->esc_like( $term ) . '%';
 		$post_types    = $include_variations ? array( 'product', 'product_variation' ) : array( 'product' );
 		$post_statuses = current_user_can( 'edit_private_products' ) ? array( 'private', 'publish' ) : array( 'publish' );
 		$type_join     = '';
 		$type_where    = '';
 		$status_where  = '';
 
+		// See if search term contains OR keywords.
+		if ( stristr( $term, ' OR ' ) ) {
```

Comment:
Should this be uppercase only? If you test with lowercase it does not work.

Reply:
Should be lower. Fixing.